base/inc/widgets/base-slider.class.php

```diff
@@ -96,6 +96,12 @@ function control_form_fields(){
 				'label' => __( 'Swipe Control', 'so-widgets-bundle' ),
 				'description' => __( 'Allow users to swipe through frames on mobile devices.', 'so-widgets-bundle' ),
 				'default' => true,
+			),
+
+			'background_video_mobile' => array(
+				'type' => 'checkbox',
+				'label' => __( 'Show Slide Background Videos on mobile', 'so-widgets-bundle' ),
```

Comment:
Should either be sentence case (_Show slide background videos on mobile_) or title case, but not a mix of the two. IMO sentence case is better.